students/HarveyHerela/lesson07/html_render.py

```diff
@@ -0,0 +1,128 @@
+class Element:
+    """Base class for all HTML tags."""
+    def __init__(self, tag_name='element', content=None, **kwargs):
+        if content is None:
+            self.content = list()
+        else:
+            self.content = [content]
+        tag_style = ''
+        if 'style' in kwargs:
+            tag_style = ' style="{style}"'.format(style=kwargs['style'])
+        self.tag_open = "<{tag}{tag_style}>".format(tag=tag_name, tag_style=tag_style)
+        self.tag_close = "</{tag}>".format(tag=tag_name)
+
+
+    def append(self, content):
+        self.content.append(content)
+
+
+    def render(self, file_out, cur_ind = "", one_line_tag=False):
+        file_out.write(cur_ind + self.tag_open)
+        if not one_line_tag:
+            file_out.write('\n')
+
+        for content in self.content:
+            if type(content) == str:
+                file_out.write(content)
+            else:
+                content.render(file_out, cur_ind + "   ")
+            if not one_line_tag:
+                file_out.write('\n')
+
+        if not one_line_tag:
+            file_out.write('\n' + cur_ind)
+        file_out.write(self.tag_close)
+
+
+class Html(Element):
+    def __init__(self, content=None, **kwargs):
+        Element.__init__(self, tag_name="html", content=content, **kwargs)
+    
```

Comment:
See my note above on making `tag_name` a class attribute, shared between all instances of these classes -- then you won't need to call `__init__()` on each `Element` subclass,  you can just override the `tag_name` attribute